cloud/ec2_test.go

```diff
@@ -757,26 +757,51 @@ func (s *EC2Suite) TestModifyHost() {
 	s.Error(s.onDemandManager.ModifyHost(ctx, s.h, changes))
 	s.Require().NoError(s.h.Remove())
 
+	s.h.CreationTime = time.Now()
+	s.h.ExpirationTime = time.Now().Add(time.Hour * 24 * 7)
+	s.h.NoExpiration = false
 	s.h.Status = evergreen.HostStopped
 	s.Require().NoError(s.h.Insert())
+
+	// updating instance tags and instance type
 	s.NoError(s.onDemandManager.ModifyHost(ctx, s.h, changes))
 	found, err := host.FindOne(host.ById(s.h.Id))
 	s.NoError(err)
-	s.Equal([]host.Tag{host.Tag{Key: "key-2", Value: "val-2", CanBeModified: true}}, found.InstanceTags)
+	s.Equal([]host.Tag{{Key: "key-2", Value: "val-2", CanBeModified: true}}, found.InstanceTags)
 	s.Equal(changes.InstanceType, found.InstanceType)
 
-	intent := host.Host{
-		Id:           "evg-1234",
-		NoExpiration: false,
+	// intent := host.Host{
+	// 	Id:             "evg-1234",
+	// 	NoExpiration:   false,
+	// 	CreationTime:   time.Now(),
+	// }
+	// s.NoError(intent.Insert())
```

Comment:
I commented this out because It didn't seem like it was being used 🤔 Can I delete it?